api/apps/api/src/modules/projects-acl/project-acl.module.ts

```diff
@@ -1,17 +1,10 @@
 import { Module } from '@nestjs/common';
 import { CqrsModule } from '@nestjs/cqrs';
-import { TypeOrmModule } from '@nestjs/typeorm';
-
-import { UsersProjectsApiEntity } from '@marxan-api/modules/projects/control-level/users-projects.api.entity';
 
 import { ProjectAclService } from './project-acl.service';
 
 @Module({
-  imports: [
-    CqrsModule,
-    // this entity most likely shouldn't be under `projects`
-    TypeOrmModule.forFeature([UsersProjectsApiEntity]),
-  ],
+  imports: [CqrsModule],
   providers: [ProjectAclService],
   exports: [ProjectAclService],
```

Comment:
Shouldn't it export `ProjectAccessControl` ?

Reply:
yeah, true, fixing!